demos/sse/app/assets/entry.tsx

```diff
@@ -0,0 +1,26 @@
+import { createFrame } from '@remix-run/dom'
+
+createFrame(document, {
+  async loadModule(moduleUrl, name) {
+    let mod = await import(moduleUrl)
+    if (!mod) {
+      throw new Error(`Unknown module: ${moduleUrl}`)
+    }
+
+    let Component = mod[name]
+    if (!Component) {
+      throw new Error(`Unknown component: ${moduleUrl}#${name}`)
+    }
+
+    return Component
+  },
+
+  async resolveFrame(frameUrl) {
```

Comment:
You shouldn't need this at all unless you're using `<Frame>`, which I didn't see anywhere. If it's an error to  omit it, we should fix that.